src/main/java/ru/ayaz/UserHandler.java

```diff
@@ -2,130 +2,42 @@
 
 import java.io.BufferedReader;
 import java.io.IOException;
-import java.io.InputStreamReader;
-import java.io.PrintWriter;
-import java.net.Socket;
 
 
 public class UserHandler implements Runnable {
```

Comment:
I think i cant initialize nickname field (using socket) cause i should ask for it first... So the logic of UserHandler as below can be broken.